src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -48,6 +50,37 @@ const defaultProps = {
 };
 
 function ReportActionItemCreated(props) {
+    const {windowWidth, isSmallScreenWidth} = useWindowDimensions();
+
+    const IMAGE_OFFSET_X = windowWidth / 2;
+    const IMAGE_OFFSET_Y = 75;
+    const ANIMATION_BOOST = 1.3;
+
+    // Get data from phone rotation sensor and prep other variables for animation
+    const animatedSensor = useAnimatedSensor(SensorType.GYROSCOPE);
+    const moveXoffset = useSharedValue(0);
+    const moveYoffset = useSharedValue(0);
+    const backgroundImageOffsetX = useSharedValue(-IMAGE_OFFSET_X);
+
+    // Apply data to create style object
+    const animatedStyles = useAnimatedStyle(() => {
+        /*
+         * We use pitch and roll instead of x and y because Reanimated makes these consistent across iOS and Android by standardizing on the iOS convention.
+         * For a visualization of what these values mean: https://howthingsfly.si.edu/flight-dynamics/roll-pitch-and-yaw
+         * These values are in radians
+         */
+        const {x, y} = animatedSensor.sensor.value;
+        // The x vs y here seems wrong but is the way to make it feel right to the user
+        moveXoffset.value = Math.min(IMAGE_OFFSET_X, Math.max(-IMAGE_OFFSET_X, moveXoffset.value - y * ANIMATION_BOOST));
```

Comment:
NABish but I think this would be a bit clearer:

1. Create a function in `NumberUtils` called `clamp`

    ```js
    /**
     * Clamp a number in a range.
     *
     * @param {Number} num
     * @param {Number} min
     * @param {Number} max
     * @returns {Number}
     */
    function clamp(num, min, max) {
        return Math.min(Math.max(num, min), max);
    }
    ```
    
1. (optional) Replace any uses of `lodashClamp` with `NumberUtils.clamp` (we're trying to move away from `lodash` and `underscore`).
1. Then use it here too:

    ```js
    xOffset.value = NumberUtils.clamp(xOffset.value - y * ANIMATION_BOOST, -IMAGE_OFFSET_Y, IMAGE_OFFSET_Y)
    ```

Reply:
One thing about it is that code inside useAnimatedStyle is run on UI thread so clamp function would need to be a worklet. To avoid confusion in such scenario I usually create function called something like `clampWorklet` so it's hard to import it without knowing it will break the app if run from JS thread.